src/app/helptext/task-calendar/cloudsync/cloudsync-form.ts

```diff
@@ -31,6 +31,11 @@ folder_tooltip: T('Enter the name of the destination folder.'),
 encryption_placeholder: T('Server Side Encryption'),
 encryption_tooltip: T('Choose <i>AES-256</i> or <i>None</i>.'),
 
+b2_chunk_size_placeholder:  T('Upload Chunk Size (MiB)'),
+b2_chunk_size_tooltip: T('Files are split into chunks of this size before upload.\
+ Up to «--transfers» chunks can be in progress at one time. The single largest file\
```

Comment:
This «--transfers» string seems to be a literal.  Is this value set in another field?  How does the user find out what the value is?

Reply:
@wblock yes, this value is set in another field, in legacy UI it's here: https://github.com/freenas/freenas/blob/master/gui/tasks/forms.py#L45

Reply:
@wblock The transfers field is already in new UI.